src/OrbitQt/ConnectToStadiaWidget.cpp

```diff
@@ -88,10 +89,23 @@ void ConnectToStadiaWidget::SetActive(bool value) {
   ui_->radioButton->setChecked(value);
 }
 
-void ConnectToStadiaWidget::Start(SshConnectionArtifacts* ssh_connection_artifacts,
-                                  std::optional<StadiaConnection> connection) {
+void ConnectToStadiaWidget::SetSshConnectionArtifacts(
+    SshConnectionArtifacts* ssh_connection_artifacts) {
```

Comment:
From an API design perspective this operation should be reversible, even if the class is currently used only in one situation in the code. The user needs some way to clear the pointer again, even though the one existing user doesn't care. So maybe add a `ClearSshConnectionArtifacts` method which sets the internal pointer to `nullptr`?

Reply:
done